monailabel/tasks/infer/basic_infer.py

```diff
@@ -267,12 +267,8 @@ def __call__(
         req = copy.deepcopy(self._config)
         req.update(request)
 
-        # model options
```

Comment:
some issue with your base version on github.. it has brought in new changes which are not related to this PR